Tests/SentryTests/Networking/TestSentryDispatchQueueWrapper.swift

```diff
@@ -24,7 +24,7 @@ class TestSentryDispatchQueueWrapper: SentryDispatchQueueWrapper {
     override func dispatch(after interval: TimeInterval, block: @escaping () -> Void) {
         dispatchAfterInvocations.record((interval, block))
         if dispatchAfterExecutesBlock {
-            block()
+            DispatchQueue.main.asyncAfter(deadline: .now() + interval, execute: .init(block: block))
```

Comment:
I needed this to actually dispatch after the interval, otherwise the test for UI event profiles always failed because this changed how the timeout dispatch worked for those kinds of traced events.